Documentation/network/bgp-control-plane.rst

```diff
@@ -286,6 +286,60 @@ io.kubernetes.service.namespace ``.meta.namespace``
 io.kubernetes.service.name      ``.meta.name``
 =============================== ===================
 
+CLI
+---
+
+There are two CLIs available to view cilium BGP peering state. One is present
+inside cilium agent. And second is cluster wide cilium-cli (Experimental).
+
+Cilium Agent CLI
+~~~~~~~~~~~~~~~~
+
+Following command will show peering status.
+
+.. code-block:: shell-session
+
+   cilium# cilium bgp peers -h
+   List state of all peers defined in CiliumBGPPeeringPolicy
```

Comment:
nit
```suggestion
   This command lists the state of all peers defined in CiliumBGPPeeringPolicy.
```

Reply:
> This is part of CLI output, unfortunately this has to be modified in CLI code. I'll leave the documentation as is, so it matches actual output.

I'm not happy with that, but I accept it as a temporary solution. 

I'll consider whether I'll accept the "sorry, it's upstream" justification in the future; otherwise, no one faces any pressure to fix bad docs.